packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -156,24 +156,22 @@ async function _getDashboardStats(app) {
   const dashboard = {}
 
   let hvSupportedVersions
+  let nHostsEol
   try {
     hvSupportedVersions = await app.apiMethods['xoa.getHVSupportedVersions']()
```

Comment:
API methods where not intended to be called from inside XO itself.

Also, calling them like that is not supported, it may work for some of them today but may break tomorrow.

To call an API method, the [`callApiMethod`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xo-mixins/api.mjs#L313) must be used (which is not easy to do because you don't have a `connection` object here).

I feel like the correct approach is to expose this method directly on the `app` object.